ext/skeleton/skeleton.c

```diff
@@ -8,6 +8,12 @@
 #include "ext/standard/info.h"
 #include "php_%EXTNAME%.h"
 
+#ifndef ZEND_PARSE_PARAMETERS_NONE
+#define ZEND_PARSE_PARAMETERS_NONE()  \
+        ZEND_PARSE_PARAMETERS_START(0, 0) \
+        ZEND_PARSE_PARAMETERS_END()
+#endif /* ZEND_PARSE_PARAMETERS_NONE */
```

Comment:
Rather than defining a compat macro, I'd prefer to switch the one use of this macro with the "old" and still perfectly fine pattern:

```
if (zend_parse_parameters_none() == FAILURE) {
    return;
}
```

Most of our internal code doesn't even use `ZEND_PARSE_PARAMETERS_NONE()`...

Reply:
I believe that Skeleton is mostly used for education purpose, so I thought that for such tutorial, using the Macros, would be easier for those who need to update the code for their own project. Am I wrong ?

Reply:
I don't care particularly strongly either way here, so I just went ahead and merged this variant.